ext/native/gfx/gl_debug_log.h

```diff
@@ -6,7 +6,7 @@
 // Utility to be able to liberally sprinkle GL error checks around your code
 // and easily disable them all in release builds - just undefine DEBUG_OPENGL.
 
-// #define DEBUG_OPENGL
+#define DEBUG_OPENGL
```

Comment:
Slipped in again.

```suggestion
// #define DEBUG_OPENGL
```

-[Unknown]

Reply:
Darn it.